src/sim.ts

```diff
@@ -0,0 +1,13 @@
+import * as Vis from './vis';
+import * as Plane from './plane';
+
+export class Sim {
+  thisVis: Vis.Vis;
+  mainPlane: plane.Plane;
```

Comment:
Alternatively if you `import {Plane} from './plane';` you can just do `mainPlane: Plane;`

I recommend this because it makes the code easier to talk about -- don't have to say things like "you need plane plane because you imported plane as plane"

Reply:
Resolved using the suggested method of importing only specified exports.